Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -138,6 +142,19 @@ u16 TMDReader::GetGroupId() const
   return Common::swap16(m_bytes.data() + offsetof(TMDHeader, group_id));
 }
 
+std::string TMDReader::GetGameID() const
+{
+  char game_id[6];
+  std::memcpy(game_id, m_bytes.data() + offsetof(TMDHeader, title_id) + 4, 4);
```

Comment:
Would it improve the readability, though? The point isn't that I'm getting a 32-bit (or 16-bit) integer, the point is that I'm getting four (or two) characters.

If I'm going to use `sizeof(TMDHeader::title_id)`, I would have to do something like `sizeof(TMDHeader::title_id) - 4`, because I only want to read half of the title ID.